index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+session_start();
+require_once('database/db.php');
+require_once('security/connexion.php');
+
+if(isTokenValid($_SESSION['token'])){
+    header("Location: /dashboard.php");
+}
+$error = false;
+$requestMethod = $_SERVER['REQUEST_METHOD'];
+
+    if($requestMethod == 'GET' && $_GET['sub'] == "true"){
```

Comment:
Warning: Undefined array key "sub" in /home/myns/Downloads/garageTrain 2/index.php on line 12

Reply:
Parameters are passed to url